selenoid.go

```diff
@@ -604,10 +621,15 @@ func defaultErrorHandler(requestId uint64) func(http.ResponseWriter, *http.Reque
 	}
 }
 
-func reverseProxy(hostFn func(sess *session.Session) string, status string) func(http.ResponseWriter, *http.Request) {
+func reverseProxy(hostFn func(sess *session.Session) string, opts ReverseProxyOpts) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
 		requestId := serial()
 		sid, remainingPath := splitRequestPath(r.URL.Path)
+
+		if opts.useOriginalPath {
+			remainingPath = r.URL.Path
```

Comment:
I need send request to browser with the same path which comes to selenoid.
It means user send bidi request to selenoid with url -> `ws://SELENOID_HOST:4444/session/SESSION_ID` and then selenoid send this to browser by url -> `ws://BROWSER_HOST:4444/session/SESSION_ID`.

But for example for devtools protocol request to browser send just for another port. For example user send req to seleoid -> `ws://SELENOID_HOST:4444/devtools/SESSION_ID` and selenoid send to browser like this -> `ws://BROWSER_HOST:7070`.

I just want to say that for BiDi protocol I don't need modify `r.URL.Path` in code below